NativeCode/Readme.md

```diff
@@ -0,0 +1,22 @@
+# QuestTest2 Custom Native Code
+
+QuestTest2 uses custom Native (C/C++) code for some tasks.
+
+This is loaded into Lua using the `ffi` library.
+
+## Build Instructions: Linux
+
+To generate project files, use premake5
+
+```sh
```

Comment:
what is this?

Reply:
in markdown, you can give code blocks (identified by three backtick marks at the start and end) with a language, to add pretty-printing.

So:
```c++
#include <iostream>
int main(int argc, char** argv)
{
    std::cout << "This is C++, starting with '```c++'" << std::endl;
    return 0;
}
```

And this is Lua:
```lua
function print_desc()
    print("This block starts with '```lua'")
end
```